src/root/Root.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react'
 import {
-    BrowserRouter as Router,
-  } from "react-router-dom";
+    HashRouter as Router,
```

Comment:
https://stackoverflow.com/questions/71984401/react-router-not-working-with-github-pages